components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -1046,63 +878,55 @@ subroutine aero_model_wetdep(dt, dlf, dlf2, cmfmc2, state,                    &!
 !
              mam_prevap_resusp_optcc = 0
 
-             if ( jnummaswtr == jaeromass ) then
+             if ( jnummaswtr == 1 ) then  ! dry mass
                    mam_prevap_resusp_optcc = 130
-             elseif ( jnummaswtr == jaeronumb .and. lphase == 1 .and. m == modeptr_coarse ) then
+             elseif ( jnummaswtr == 0 .and. lphase == 1 .and. imode == modeptr_coarse ) then ! number
```

Comment:
Constants like 130 and 120 should also be all caps parameters with descriptive names so that it is easy to know what they mean.